app/models/user.rb

```diff
@@ -51,16 +52,17 @@ def self.search(term, role)
   end
 
   # notifications
-  def notify(title, message, link = '')
-    notify_slack("*#{title}*\n#{message}\n#{link}")
+  def notify(title, message, link = '', type: :default)
+    last_notification = notifications.first
```

Comment:
not `notifications.last`?

Reply:
`notifications.first` is correct, Notifications are ordered by descending created_at order.

See models/notification.rb:
``` ruby
default_scope { order(read: :asc, created_at: :desc) }
```
So notifications with the newest ("largest") timestamp are first

Reply:
Thanks for the info!

Reply:
https://github.com/hpi-swt2/vm-portal/pull/529/commits/2dd51a1a95e24e7b76abbf80c54c1d7aedec4e49